src/index.html

```diff
@@ -5,8 +5,302 @@
+          <p class="card__type">Product code: 195434</p>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </div>
+          <a
+            href="#buy"
+            class="card__buy-link"
+            data-qa="card-hover"
+          >
+          buy
+          </a>
+        </div>
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            class="card__img"
+            alt="APPLE A1419 iMac 27"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__type">Product code: 195434</p>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </div>
+          <a
+            href="#buy"
+            class="card__buy-link"
+            data-qa="hover"
+          >
+          buy
+          </a>
+        </div>
+        <div class="card" data-qa="card">
+          <img
+          src="./images/imac.jpeg"
+          class="card__img"
+          alt="APPLE A1419 iMac 27"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__type">Product code: 195434</p>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <p class="card__price-label">Price:</p>
+            <p class="card__price-value">$2,199</p>
+          </div>
+          <a
+          href="#buy"
+          class="card__buy-link"
+          data-qa="hover"
+          >
+          buy
+          </a>
```

Comment:
Need to fix it everywhere
```suggestion
          <a
            href="#buy"
            class="card__buy-link"
            data-qa="hover"
          >
            buy
          </a>
```